test/jasmine/tests/annotations_test.js

```diff
@@ -721,8 +721,8 @@ describe('annotations autorange', function() {
                 text: 'LT',
                 x: -1,
                 y: 3,
-                xref: 'x5', // will be converted to 'x' and xaxis should autorange
-                yref: 'y5', // same 'y' -> yaxis
+                xref: 'xq', // will be converted to 'x' and xaxis should autorange
+                yref: 'yz', // same 'y' -> yaxis
```

Comment:
Why did you change this?

Reply:
> Why did you change this?

Now an annotation is enough to create a new subplot, so I had to switch these to invalid references.